src/sonic-frr/Makefile

```diff
@@ -0,0 +1,24 @@
+.ONESHELL:
+SHELL = /bin/bash
+.SHELLFLAGS += -e
+
+MAIN_TARGET = frr_$(FRR_VERSION)-dev_amd64.deb
+
+$(addprefix $(DEST)/, $(MAIN_TARGET)): $(DEST)/% :
+        # Cloning FRR repo if not already done
+	if [ -d "~/frr" ]; then \
+		git clone https://github.com/FRRouting/frr.git; \
```

Comment:
do you want to checkout a tag or this master branch, in this case the nightly build will be different as the frr moves forward.